src/components/Icon/Icon.type.ts

```
@@ -0,0 +1,23 @@
+export type IconColor = ColorKeys;
+
+export interface IconStyleProps {
+  iconColor?: IconColor;
+  iconType: IconType;
+}
+
+export interface IconStylePropsWithTheme extends IconStyleProps {
+  theme: Theme;
+}
+
+export interface IconCustomProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)